install/kubernetes/helm/istio/charts/grafana/templates/service.yaml

```diff
@@ -15,5 +15,4 @@ spec:
       protocol: TCP
       name: {{ .Values.service.name }}
   selector:
-    app: {{ template "grafana.name" . }}
-    release: {{ .Release.Name }}
```

Comment:
same comment here...

Reply:
Confirmed that the release/etc show up in the 'deployment' (as labels). 
The change only removes them from the replica set and pods.